main.tf

```diff
@@ -14,65 +14,77 @@
  * limitations under the License.
  */
 
-terraform {
-  required_version = "~> 0.11.0"
-}
-
 locals {
-  prefix = "${var.prefix == "" ? "" : "${var.prefix}-"}${lower(var.location)}"
+  prefix = var.prefix == "" ? "" : join("-", list(var.prefix, ""))
 }
 
 resource "google_storage_bucket" "buckets" {
-  count              = "${length(var.names)}"
-  name               = "${var.prefix}-${lower(element(var.names, count.index))}"
-  project            = "${var.project_id}"
-  bucket_policy_only = "${lookup(var.bucket_policy_only, lower(element(var.names, count.index)), true)}"
-  location           = "${var.location}"
-  storage_class      = "${var.storage_class}"
-  labels             = "${var.labels}"
-
+  count         = length(var.names)
+  name          = "${local.prefix}${lower(element(var.names, count.index))}"
+  project       = var.project_id
+  location      = var.location
+  storage_class = var.storage_class
+  labels        = var.labels
+  force_destroy = lookup(
```

Comment:
This looks like an unrelated change?

Reply:
You mean force destroy? Yep, forgot I had added it. :) I think it's really needed, it can of course go in a separate PR if you prefer.

Reply:
Fine to keep it here, just please add a note to the CHANGELOG.

Reply:
Added note.